### DISCUSSION TITLE
ScriptReference/Rendering.CommandBuffer.Blit

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#695] [Notes]
# ScriptReference/Rendering.CommandBuffer.Blit

https://docs.unity3d.com/ScriptReference/Rendering.CommandBuffer.Blit.html

**Comment by naelstrof** [comment]
This command can attempt to render the full-screen quad inside-out if the previous command was a DrawRenderer() with an inside-out renderer.
Ensure `Cull Off` is present in the shader you use for the full-screen blit.
